### PR TITLE
Add examples and WIP warnings

### PR DESCRIPTION
**Pull request by florian-lefebvre**
- add "Work in progress" hints on each empty page
- in `getting-started/examples`, put same examples as on https://web3forms.com/examples
- add a Nuxt example using `@nuxtjs/axios` module
- update summary

### CODE REVIEW

File: getting-started/pro-features/add-cc-email.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # Add CC Email
 
+{% hint style="info" %}
+Heads Up! This is a PRO feature. You must have an active membership to use this feature.
+{% endhint %}
+
+{% hint style="info" %}
+COMING SOON !
+{% endhint %}
```
Comment (surjithctly) [inline getting-started/pro-features/add-cc-email.md]: `ccemail` is not coming soon. its already available via adding `ccemail` with a `hidden` attribute.

```html
  <input type="hidden" name="ccemail" value="partner@example.com">
```

File: getting-started/pro-features/file-attachments.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
 # File Attachments
 
+{% hint style="info" %}
+Heads Up! This is a PRO feature. You must have an active membership to use this feature.
+{% endhint %}
+
+{% hint style="info" %}
+COMING SOON !
+{% endhint %}
```
Comment (surjithctly) [inline getting-started/pro-features/file-attachments.md]: File attachments is also available using `<input type="file" name="attachment">`

File: getting-started/pro-features/webhooks.md [hunk 3]
```diff
@@ -1,2 +1,9 @@
 # Webhooks
 
+{% hint style="info" %}
+Heads Up! This is a PRO feature. You must have an active membership to use this feature.
+{% endhint %}
+
+{% hint style="info" %}
+COMING SOON !
+{% endhint %}
```
Comment (surjithctly) [inline getting-started/pro-features/webhooks.md]: Webhooks is also available using `<input type="hidden" name="webhook" value="__URL__">`

**Review by surjithctly — CHANGES_REQUESTED** [review]
Thank you for your contribution.

Could you make the mentioned changes?

**Comment by florian-lefebvre** [comment]
Also, should I update the API reference with those pro features ?

**Comment by surjithctly** [comment]
Yeah sure. Go ahead!

**Comment by florian-lefebvre** [comment]
It's done, I think I have nothing else to add to this pull request.

**Comment by surjithctly** [comment]
Thanks man!

**Pull request merged**